maps_dashboards/public/components/map_page/map_page.tsx

```diff
@@ -0,0 +1,56 @@
+/*
+ * Copyright OpenSearch Contributors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import React from 'react';
+import { i18n } from '@osd/i18n';
+import { AppMountParameters, CoreStart } from 'opensearch-dashboards/public';
+import {
+  NavigationPublicPluginStart as NavigationStart,
+  TopNavMenuData,
+} from '../../../../../src/plugins/navigation/public';
+import { PLUGIN_ID } from '../../../common';
+import { MapContainer } from '../map_container';
+
+interface MapContainerProps {
+  navigation: NavigationStart;
+  setHeaderActionMenu: AppMountParameters['setHeaderActionMenu'];
+  savedObjectsClient: CoreStart['savedObjects']['client'];
+}
+
+export const MapPage = ({
+  navigation,
+  setHeaderActionMenu,
+  savedObjectsClient,
+}: MapContainerProps) => {
+  const topNavMenuConfigs: TopNavMenuData[] = [];
```

Comment:
nit: I'd move creating this config into its own function for easier testing, but that just personal preference

Reply:
Yes, separated out the configs to `get_top_nav_config.tsx`